HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -4688,6 +4688,26 @@ Patch "Infinite Quick Cyloop Gauge" in "Cheats" by "NM"
     );
 }
 
+Patch "Play Monologues Unconditionally" in "Audio" by "WasifBoomz" does
```

Comment:
If all this code does is allow monologues to play outside of heightmap terrain, then it'd be better to name the code after something that benefits the game with this change.

A better name would be "Disable Monologue Restrictions".